Modules/vc-module-search/VirtoCommerce.SearchModule.Web/module.manifest

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<module>
+    <id>VirtoCommerce.Search</id>
+    <version>3.0.0</version>
+    <platformVersion>3.0.0</platformVersion>
+    <title>Search module</title>
+    <description>Search module</description>
+    <authors>
+        <author>Eugeny Tatarincev</author>
+    </authors>
+    <owners>
+        <owner>Virto Commerce</owner>
+    </owners>
+    <projectUrl>https://virtocommerce.com/apps/extensions/virto-core-module</projectUrl>
+    <iconUrl>Modules/$(VirtoCommerce.Search)/Content/logoVC.png</iconUrl>
+    <releaseNotes></releaseNotes>
+    <copyright>Copyright © 2011-2018 Virto Commerce. All rights reserved</copyright>
+    <tags>security core</tags>
+
+    <assemblyFile>VirtoCommerce.SearchModule.Web.dll</assemblyFile>
+    <moduleType>VirtoCommerce.SearchModule.Web.Module, VirtoCommerce.SearchModule.Web</moduleType>
+
+    <styles>
+        <directory virtualPath="$/Content" searchPattern="*.css" searchSubdirectories="true" />
+    </styles>
+    <scripts>
+        <directory virtualPath="$/Scripts" searchPattern="*.js" searchSubdirectories="true" />
+    </scripts>
+    
+    <incompatibilities>
+    </incompatibilities>
+    
+    <settings>
```

Comment:
remove settings from module manifest, use from code registration instead (see the platform settings registration)